packages/frontend/core/src/components/blocksuite/block-suite-editor/specs/custom/root-block.ts

```diff
@@ -107,7 +107,9 @@ function createThemeExtension(framework: FrameworkProvider) {
       if (cache) return cache;
 
       const appTheme$ = framework.get(AppThemeService).appTheme.theme$;
-      const docTheme$ = doc.properties$.map(props => props.edgelessColorTheme);
+      const docTheme$ = doc.properties$.map(
+        props => props.edgelessColorTheme || 'system'
```

Comment:
fallback to appTheme$ instead?

Reply:
This default value logic is also used in UI components, making the use of `edgelessColorTheme || 'system'` more consistent here.